webpack.config.js

```diff
@@ -125,7 +126,16 @@ module.exports = function(env = {}, options = {}) {
       new ExtractTextPlugin({
         filename: assetsSubDirectory + `css/${prefix}[name].css?[chunkhash]`, // NOTE: postcss-pipeline currently does not support query hash (https://github.com/mistakster/postcss-pipeline-webpack-plugin/issues/30)
         allChunks: true,
-      })
+      }),
+
+      // Cleans dist directory and removes specific unnecessary files
+      new WebpackCleanPlugin([
```

Comment:
- [ ] Please base theme array on `themes`:

```
new WebpackCleanPlugin(
  Object.keys(themes).map(theme => `${theme}.js`),
  { basePath: path.join(__dirname, 'dist/static/js') }),
);
```

Even better would be to move the setup to the beginning of the config where the variables are defined:

```
const clean = [
  ...Object.keys(themes).map(theme => `${theme}.js`),
  'any/other/files.js',
]
```